### PR TITLE
Add email api, ac summary email template, instructor email template and part of ranking status page

### PR DESCRIPTION
**Pull request by j2kan**

### CODE REVIEW

**Review by smwatts — DISMISSED** [review]
Thanks for changing how you assign your variables in the email template!

File: taworks/taform/views.py [hunk 1]
```diff
@@ -19,6 +19,120 @@
...
+            https://team4.uwaterloo.ca/taform/instructor/{url}</div>
+            <br/>
+            <div>Regards,</div>
+            <div>Associate Chair for Undergraduate Studies, Management Sciences
+            </div>
+            <br/>
+            <div>*Note: If you are the Instructor for more than one course in the 
+            upcoming term, you will receive an email for each course. Important: 
+            Links are specific to each class.
+            You may also need to be on campus wifi or vpn if you are remote.</div>
```
Comment (amyleblond) [inline taworks/taform/views.py]: can we make this bold? Just to make it stand out more -- just my opinion but good either way!

File: taworks/taform/views.py [hunk 2]
```diff
@@ -42,7 +156,8 @@ def apply(request):
     if request.method == 'POST':
         num = [x for x in models.Course.objects.all()]
         s_form = models.StudentForm(request.POST, request.FILES or None)
-        a_forms = [models.ApplicationForm(request.POST, prefix=str(x), instance=models.Application()) for x in range(len(num))]
+        a_forms = [models.ApplicationForm(request.POST, prefix=str(x),
```
Comment (amyleblond) [inline taworks/taform/views.py:159]: just making length of line smaller?

**Review by amyleblond — DISMISSED** [review]
Just had one potential change of making the "*Note:" on instructor email bold or more prominent. Not necessary though!

Great job!

File: taworks/taform/views.py (hunk 1 above)
Comment (j2kan) [reply taworks/taform/views.py]: Done

File: taworks/taform/views.py (hunk 2 above)
Comment (j2kan) [reply taworks/taform/views.py:159]: Yep, try to keep it 80 lines whenever possible.

**Review by amyleblond — APPROVED** [review]
thanks for bold

**Comment by j2kan** [comment]
got lgtm from @amyleblond  and @smwatts

**Pull request merged**